cartography/intel/github/repos.py

```diff
@@ -461,7 +466,7 @@ def load_github_owners(neo4j_session: neo4j.Session, update_tag: int, repo_owner
             WITH user
 
             MATCH (repo:GitHubRepository{id: $RepoId})
-            MERGE (user)<-[r:OWNER]-(repo)
+            MERGE (user)-[r: REPOSITORY]->(repo)
```

Comment:
let's not change the relationships. this will create a problem when we sync up with upstream